integration-templates/xero/mappers/to-invoice.ts

```diff
@@ -86,45 +86,47 @@ export function toXeroInvoice(invoice: CreateInvoice | Invoice) {
         xeroInvoice['Url'] = invoice.url;
     }
 
-    for (const item of invoice.fees) {
-        const xeroItem: Record<string, any> = {
-            Description: item.description,
-            AccountCode: item.account_code
-        };
+    if (invoice.fees) {
```

Comment:
is the `if` necessary?

Reply:
Compilation complains if the conditional isn't there
```
xero/mappers/to-invoice.ts:89:28 - error TS18048: 'invoice.fees' is possibly 'undefined'.

89         for (const item of invoice?.fees) {
```

Reply:
what about ` for (const item of invoice?.fees || []) {`